Framework/Algorithms/src/CalculatePlaczekSelfScattering.cpp

```diff
@@ -112,7 +116,7 @@ void CalculatePlaczekSelfScattering::exec() {
   std::vector<double> eps1;
   constexpr auto LambdaD = 1.44;
```

Comment:
should we move this constexpr into the anonymous namespace above?  Also, I am kind of curious where does this 1.44 comes from :thinking: